packages/SwingSet/src/kernel/vat-loader/manager-nodeworker.js

```diff
@@ -64,7 +64,7 @@ export function makeNodeWorkerVatManagerFactory(tools) {
 
     function sendToWorker(msg) {
       assert(msg instanceof Array);
-      workerP.then(worker => worker.postMessage(msg));
+      void workerP.then(worker => worker.postMessage(msg));
```

Comment:
ok, I don't even know how that could fail, also we don't love `manager-nodeworker.js` anyways